community/samples/serving/helloworld-python-bentoml/README.md

```diff
@@ -0,0 +1,243 @@
+---
+title: "Hello World - Python BentoML"
+linkTitle: "Python Bentoml"
+weight: 1
+type: "docs"
+---
+
+A simple machine learning model with API serving that is written in python and
+using [BentoML](https://github.com/bentoml/BentoML). BentoML is an open source
+framework for high performance ML model serving, which supports all major machine
+learning frameworks including Keras, Tensorflow, PyTorch, Fast.ai, XGBoost and etc.
+
+This sample will walk you through the steps of creating and deploying a machine learning
+model using python. It will use BentoML to package a classifier model trained
+on the Iris dataset. Afterward, it will create a container image and
+deploy the image to Knative.
+
+Knative deployment guide with BentoML is also available in the
+[BentoML documentation](https://docs.bentoml.org/en/latest/deployment/knative.html)
+
+## Before you begin
+
+- A Kubernetes cluster with Knative installed. Follow the
+  [installation instructions](../../../../docs/install/README.md) if you need to
+  create one.
+- [Docker](https://www.docker.com) installed and running on your local machine,
+  and a Docker Hub account configured. Docker Hub will be used for a container registry).
+- Python 3.6 or above installed and running on your local machine.
+  - `scikit-learn` and `bentoml` packages, run command:
+
+    ```shell
+    pip install scikit-learn
+    pip install bentoml
+    ```
+
+## Recreating sample code
```

Comment:
Put some comment in here about running all of these steps locally, just so it's clear that these don't need a kubernetes cluster.